src/main/resources/assets/js/app/inputtype/ui/text/HtmlEditor.ts

```diff
@@ -855,6 +855,14 @@ class HtmlEditorConfigBuilder {
         return 'resize,image';
     }
 
+    private getExtraAllowedContent(): string {
+        if (this.editorParams.isScriptAllowed()) {
+            return 'code address dl dt dd blockquote script;img[data-src]';
+        }
+
+        return 'code address dl dt dd blockquote;img[data-src]';
+    }
```

Comment:
This is a bit lazy :)
We shouldn't duplicate the entire string here. Just create a ``const`` that will be either `` script`` or empty string depending on ``this.editorParams.isScriptAllowed()`` and then use that const inside the string as a template literal:
```
return `code address dl dt dd blockquote${scriptTag};img[data-src]`;
```

Reply:
He-he, ok :)